modules/serverless-rn-1-25-0.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies
+//
+// * serverless/serverless-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="serverless-rn-1-25-0_{context}"]
+= Release notes for Red Hat {ServerlessProductName} 1.25.0
+
+{ServerlessProductName} 1.25.0 is now available. New features, changes, and known issues that pertain to {ServerlessProductName} on {product-title} are included in this topic.
+
+[id="new-features-1.25.0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 1.4.
+* {ServerlessProductName} now uses Knative Eventing 1.4.
+* {ServerlessProductName} now uses Kourier 1.4.
+* {ServerlessProductName} now uses Knative (`kn`) CLI 1.4.
+* {ServerlessProductName} now uses Knative Kafka 1.4.
+* The `kn func` CLI plug-in now uses `func` 1.7.0.
+
+* {FunctionsProductName} is now generally available (GA). Only the Quarkus runtime is generally available. The other provided runtimes are available as a Technology Preview (TP). See xref:../serverless/functions/serverless-functions-getting-started.adoc#serverless-functions-getting-started[Getting started with functions].
+* Integrated development environments (IDE) plug-ins for creating and deploying functions are now available for link:https://github.com/redhat-developer/vscode-knative[Visual Studio Code] and link:https://github.com/redhat-developer/intellij-knative[IntelliJ].
+* Knative Kafka Broker is now generally available (GA). Knative Kafka Broker is a highly performant implementation of the Knative Broker API, directly targeting Apache Kafka.
```

Comment:
As part of the GA for Broker and KafkaSink, did we remove the TP notes in docs, like at page https://docs.openshift.com/container-platform/4.7/serverless/admin_guide/serverless-kafka-admin.html 
```
Kafka broker (Technology Preview)

Kafka sink (Technology Preview)
```
or at this section https://docs.openshift.com/container-platform/4.7/serverless/develop/serverless-using-brokers.html#serverless-using-brokers-kafka_serverless-using-brokers

Reply:
(there might be other instances I haven't checked all pages)